apis/python/CMakeLists.txt

```diff
@@ -0,0 +1,332 @@
+cmake_minimum_required(VERSION 3.21)
+cmake_policy(SET CMP0148 NEW) 
```

Comment:
Why does the policy need to be set to `NEW`? It doesn't change anything if we don't use a deprecated find module, does it?